packages/pages/src/components/link/link.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import classNames from "classnames";
+import { getHref, isEmail, reverse } from "./methods";
+import type { CTA } from "./types";
+
+/**
+ * Configuration options available for any usages of the Link component.
```

Comment:
Add `@public` to any docs that should be shown to users. It allows the docs to output in the doc generation.